lib/localparticipant.js

```diff
@@ -438,7 +438,7 @@ class LocalParticipant extends Participant {
     stop = typeof stop === 'boolean' ? stop : true;
     return this.unpublishTracks(tracks).map(publication => {
       const track = publication.track;
-      if (stop) {
+      if (stop && track.kind !== 'data') {
```

Comment:
Nice, this is what I expected we needed to change 👍 Can you also include a test validating that we do not regress in this particular area, or does your test change below cover that?

Reply:
We need to also apply this fix in removeTrack as well. Making changes